# HG changeset patch
# User Rik <rik@octave.org>
# Date 1381015861 25200
#      Sat Oct 05 16:31:01 2013 -0700
# Node ID 7af442c326a85b7261020fbaa7a5ce4ddac7309e
# Parent  387ecd448b30d69498bfc6fba3ef92ac8e94ffcb
daspect.m: Remove unnecessary () enclosing code sample in docstring.

* scripts/plot/appearance/daspect.m: Remove unnecessary () enclosing code
sample in docstring.

diff --git a/scripts/plot/appearance/daspect.m b/scripts/plot/appearance/daspect.m
--- a/scripts/plot/appearance/daspect.m
+++ b/scripts/plot/appearance/daspect.m
@@ -22,17 +22,17 @@
 ## @deftypefnx {Function File} {} daspect (@var{mode})
 ## @deftypefnx {Function File} {@var{data_aspect_ratio_mode} =} daspect ("mode")
 ## @deftypefnx {Function File} {} daspect (@var{hax}, @dots{})
 ## Query or set the data aspect ratio of the current axes.
 ##
 ## The aspect ratio is a normalized 3-element vector representing the span of
 ## the x, y, and z-axis limits.
 ## 
-## @code{(daspect (@var{mode}))}
+## @code{daspect (@var{mode})}
 ##
 ## Set the data aspect ratio mode of the current axes.  @var{mode} is
 ## either @qcode{"auto"} or @qcode{"manual"}.
 ## 
 ## @code{daspect (@qcode{"mode"})}
 ##
 ## Return the data aspect ratio mode of the current axes.
 ## 
